packages/bbui/src/Tabs/Tabs.svelte

```diff
@@ -98,7 +101,9 @@
   .quiet {
     border-bottom: none !important;
   }
-
+  .onTop {
+    z-index: 1;
```

Comment:
Is this z-index going to be high enough consistently? If its for dropdowns etc it might need to be something like `20`.

Reply:
Sounds good, added in https://github.com/Budibase/budibase/pull/3932/commits/c0006b4d8f6ff1d5d03734d71a5b94e57d84db60